site/pages/dashboard.page.tsx

```diff
@@ -262,6 +262,9 @@ const Dashboard = ({
                     <h2 className="govuk-heading-s text-govBlue">Templates</h2>
                 </Link>
             )}
+            <Link className="govuk-link" href="/roadworks-overview">
```

Comment:
We could wrap this in a feature flag so it doesn't show up on prod

Reply:
done